builtins/amp-img.js

```diff
@@ -200,7 +200,8 @@ export class AmpImg extends BaseElement {
       return;
     }
     // No need to generate sizes if already present.
-    const sizes = this.element.getAttribute('sizes');
+    const sizes =
```

Comment:
Is getAttribute used here in case the attribute value is empty, in which case hasAttribute would return true but getAttribute would be falsey?

Reply:
There was no test for this, but we could add one? 

Right now I am not certain what the values should be in those cases.